src/vicsetup/libvicsetup/crypto.c

```diff
@@ -0,0 +1,388 @@
+#include <string.h>
+#include <sys/time.h>
+#include <assert.h>
+#include <stdlib.h>
+
+#include <mbedtls/pkcs5.h>
+#include <mbedtls/aes.h>
+#include <mbedtls/cipher.h>
+#include <mbedtls/sha1.h>
+#include <mbedtls/sha256.h>
+#include <mbedtls/sha512.h>
+#include <mbedtls/base64.h>
+#include <mbedtls/entropy.h>
+#include <mbedtls/ctr_drbg.h>
+#include <pthread.h>
+
+#include "../../../third_party/argon2/include/argon2.h"
```

Comment:
Why don't you add argon2 to the include path and simply `#include "argon2.h"`?

Reply:
Fixed. There used to be two argon2.h headers but the local one has been deleted.